esti/lakectl_util.go

```diff
@@ -31,7 +31,7 @@ var (
 	reShortCommitID   = regexp.MustCompile(`[\d|a-f]{16}`)
 	reChecksum        = regexp.MustCompile(`([\d|a-f]{32})|(0x[0-9A-F]{15})`)
 	reEndpoint        = regexp.MustCompile(`https?://\w+(:\d+)?/api/v\d+/`)
-	rePhysicalAddress = regexp.MustCompile(`/data/[0-9a-v]{20}/[0-9a-v]{20}?(--sig--.+)*`)
+	rePhysicalAddress = regexp.MustCompile(`/data/[0-9a-v]{20}/[0-9a-v]{20}?(,.+)?`)
```

Comment:
To quote several wise people,
> Now we have two problems.

We don't need to capture, so I'm changing to non-capturing brackets.  But this expression also allows e.g. `/data/12345678901234567890/,checksum`, which seems wrong.  I think what you _really_ want is probably:
```suggestion
	rePhysicalAddress = regexp.MustCompile(`/data/[0-9a-v]{20}/(?:[0-9a-v]{20}(?:,.+)?)?`)
```

Reply:
True, but I didn't want to touch that part because I wasn't sure why it was written that way and if there's some legitimate reason for it